### ISSUE TITLE
Python version error ( add support for 3.7)

### ISSUE THREAD

**Issue by gauthamzz** [ucbrise/clipper#580]
` ClipperException: Python closure deployer only supports Python 2.7, 3.5, and 3.6. Detected 3.7 `

when excecuting the basic example 
```
 python_deployer.deploy_python_closure(clipper_conn, name="sum-model", version=1, input_type="doubles", func=feature_sum) 
   ...:                                                                                                                                                
18-10-01:18:04:52 INFO     [deployer_utils.py:44] Saving function to /tmp/clipper/tmp0rq77bb7
18-10-01:18:04:52 INFO     [deployer_utils.py:54] Serialized and supplied predict function
18-10-01:18:04:52 INFO     [python.py:192] Python closure saved
18-10-01:18:04:52 ERROR    [python.py:214] Python closure deployer only supports Python 2.7, 3.5, and 3.6. Detected 3.7
---------------------------------------------------------------------------
ClipperException                          Traceback (most recent call last)
<ipython-input-8-f507994c395f> in <module>
----> 1 python_deployer.deploy_python_closure(clipper_conn, name="sum-model", version=1, input_type="doubles", func=feature_sum)

~/hacks/thesis/clipper/venv/lib/python3.7/site-packages/clipper_admin/deployers/python.py in deploy_python_closure(clipper_conn, name, version, input_type, func, base_image, labels, registry, num_replicas, batch_size, pkgs_to_install)
    215             # Remove temp files
    216             shutil.rmtree(serialization_dir)
--> 217             raise ClipperException(msg)
    218 
    219     # Deploy function

ClipperException: Python closure deployer only supports Python 2.7, 3.5, and 3.6. Detected 3.7
```

Python version 3.7.0
OS Manjaro Linux

**Comment by simon-mo** [comment]
@gauthamzz Thanks for the report! We are aware of it. Python3.7 will be added very soon

**Comment by gauthamzz** [comment]
Is there a workaround. Other than changing Python version for whole system?
Anything with virtualenv or something ?

**Comment by simon-mo** [comment]
If you run your code under virtualenv it should be choosing the corresponding version.

**Comment by rkooo567** [comment]
@gauthamzz I believe @withsmilo will handle this. I will create an issue for this.

**Comment by withsmilo** [comment]
resolved by #723
